esp/templates/program/modules/creditcardmodule_stripe/failure.html

```diff
@@ -11,19 +11,17 @@
 {% endblock %}
 
 {% block content %}
-<h1>Credit Card Payment for {{ program.niceName }}</h1>
+<div style="background-color:#D3D3D3;">
```

Comment:
Can we put these in a stylesheet instead?

Reply:
Yes. I am a hack :P

Reply:
Further evidence: would it be better to make a new stylesheet or put it in an existing one (the only ones that are loaded are theme-controlled)

Reply:
Put it in an existing one, probably.  One day we will clean up our CSS maybe, but that day is not today.

Reply:
I'm sorry I still don't understand the structure of how these things get loaded. The only stylesheet that appears to be loaded in the final page is theme_compiled.css, and I don't know which sheets go into that one. Could you explain a little more please?

Reply:
I think it loads everything in `esp/public/media/theme_editor/less/`.

Reply:
So you would just pick the most fitting `.less` file in there and include your new CSS in that file. Then it will be included in `theme_compiled.css` the next time the website applies/recompiles a theme.